src/main/java/com/kodilla/ecommercee/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package com.kodilla.ecommercee.exception;
+
+import com.kodilla.ecommercee.domain.UserDto;
+
+public class UserNotFoundException extends Exception {
+
+    private UserDto nullUser = new UserDto(null, null, null,false, null);
```

Comment:
czy to jest dobra praktyka tworzyć pustego użytkownika w momencie wystąpienia błędu? Nie lepiej wyświetlić info i przerwać operację?

Reply:
Lepiej wyrzucić po prostu sam wyjątek, a nie tworzyć użtkownika